ci.R

```diff
@@ -0,0 +1,184 @@
+## most of functions from R tools4pkgs branch
+## https://github.com/wch/r-source/tree/tools4pkgs
+## https://svn.r-project.org/R/branches/tools4pkgs/src/library/tools/R/packages.R
```

Comment:
Yes, but all that code was written by me and pushed to R tools4pkgs branch. Hope this solves that problem.

Reply:
It might be better to reword the header so that it says
```
## most of the functions here were also contributed to R `tools4pkgs` branch
```
to avoid the appearance of any impropriety.